src/pages/offer-details/OfferDetails.jsx

```diff
@@ -1,7 +1,49 @@
+import { useState, useEffect } from 'react'
 import PageWrapper from '~/components/page-wrapper/PageWrapper'
+import HeaderCard from '~/components/header-card/HeaderCard'
+import AboutOffer from '~/components/about-offer/AboutOffer'
+import offerIcon from '~/assets/img/offer-page/offer-icon.svg'
+import { useSelector } from 'react-redux'
+import { useLocation } from 'react-router-dom'
+import { offerService } from '~/services/offer-service'
 
 const OfferDetails = () => {
-  return <PageWrapper>OfferDetails Page</PageWrapper>
+  const { pathname } = useLocation()
+  const codedOfferId = pathname.split('/')[2]
```

Comment:
Use [`useParams`](https://reactrouter.com/6.29.0/hooks/use-params) when working with URL params:
```javascript
const { id } = useParams()
```

Keep in mind that the `id` is our own name and depends on how we name it in the [route](https://github.com/iva-stasia/s2s-frontEnd/blob/156b34cc9383e865911bceeb50f3dfa68e81eb87/src/router/constants/authRoutes.js#L14).